disnake/invite.py

```diff
@@ -295,7 +295,7 @@ class Invite(Hashable):
     +------------------------------------+---------------------------------------------------------------------+
     | :attr:`approximate_presence_count` | :meth:`Client.fetch_invite` with ``with_counts`` enabled            |
     +------------------------------------+---------------------------------------------------------------------+
-    | :attr:`expires_at`                 | :meth:`Client.fetch_invite` with ``with_expiration`` enabled        |
+    | :attr:`expires_at`                 | :meth:`Client.fetch_invite` always included                         |
     +------------------------------------+---------------------------------------------------------------------+
```

Comment:
```suggestion
```
Looks like this is always included everywhere now? In which case this attribute can be removed from the table entirely.